config/packages/twig.yaml

```diff
@@ -9,4 +9,5 @@ twig:
         web_to_case_post_url: 'https://crowncommercial--PreProd.cs102.my.salesforce.com/servlet/servlet.WebToCase?encoding=UTF-8'
         web_to_case_orgid: '00D1j0000008aYA'
         web_to_case_record_type: '012b00000005NWC'
-        web_to_case_debug_email: 'carlos@studio24.net'
\ No newline at end of file
+        web_to_case_debug_email: 'carlos@studio24.net'
```

Comment:
What is Carlos doing here?

Reply:
This is always been here and also this is the config file for local environment.